test/testData/generate/getSwapPath.ts

```diff
@@ -6,54 +6,56 @@ import {
     type ExactInQueryOutput,
     type ExactOutQueryOutput,
 } from '@balancer/sdk';
+import { TransformBigintToString } from '../types';
 
-export type SwapPathInput = {
-    swapKind: SwapKind;
+export type Path = {
     pools: {
         poolAddress: Address;
         poolType: string;
     }[];
     tokens: Address[];
     amountRaw: bigint;
+    calculatedAmountRaw: bigint;
+};
+
+export type SwapPathInput = {
+    swapKind: SwapKind;
+    paths: Path[];
 };
```

Comment:
Adapted integration tests to accept multiple paths as input, instead of the previous solution that accepted only a single path. Not integration tests are able to test against split paths. This makes a lot easier to debug splitPath scenarios reported by users.
Ps: this is what enabled investigating split path issues, which werre found and fixed with `mutateBalances` set to true